web/src/components/map/utils.js

```diff
@@ -0,0 +1,50 @@
+import _ from 'lodash';
+
+import {
+  BLOCKS_ACROSS,
+  BLOCKS_DOWN,
+  DEFAULT_BLOCK,
+  EMPTY,
+  VISIBLE_BLOCKS_ACROSS,
+} from './constants';
+
+// converts row and col to position in 1d array
+export function getBlockIndex(row, col) {
+  return row * VISIBLE_BLOCKS_ACROSS + col;
+}
+
+// TODO: have this run once total, not once per client - not sure how to do this
```

Comment:
Eventually we will probably have this for free because the server will send the “initial” map state to clients when they join; it won’t be clients’ responsibility to come up with the map by themselves.